lxd/device/device_interface.go

```diff
@@ -16,14 +16,19 @@ type VolatileSetter func(map[string]string) error
 // and should remove the prefix before being returned.
 type VolatileGetter func() map[string]string
 
-// Device represents a device that can be added to an instance.
-type Device interface {
+// Type represents an LXD network driver type.
```

Comment:
Hmm, what? :)

Reply:
`Type represents a LXD device type.`

(Also reminder that's it `a LXD something` not `an LXD something` because of how LXD is meant to be pronounced ;)

Reply:
I'll do this update myself since it's late for you.

Reply:
Ah yeah cheers